jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpSenderOverHTTP.java

```diff
@@ -233,6 +235,8 @@ protected Action process() throws Exception
                             chunkBuffer == null ? -1 : chunkBuffer.remaining(),
                             contentBuffer == null ? -1 : contentBuffer.remaining(),
                             result, generator);
+                httpConnectionOverHTTP.addBytesOut( BufferUtil.length( headerBuffer ) + BufferUtil.length( contentBuffer ));
+
```

Comment:
I think the chunkBuffer should be included, as they too are bytes written and can be significant if chunks are small.

Reply:
Also this is the wrong spot for the addByteOut call, it should be in the FLUSH case below.